src-docs/src/views/datagrid/column_cell_actions.js

```diff
@@ -0,0 +1,152 @@
+import React, { useState, useCallback } from 'react';
+import { fake } from 'faker';
+
+import { EuiDataGrid, EuiAvatar } from '../../../../src/components/';
+
+const columns = [
+  {
+    id: 'avatar',
+    initialWidth: 40,
+    isResizable: false,
+    actions: false,
+  },
+  {
+    id: 'name',
+    initialWidth: 100,
+    isSortable: true,
+    actions: {
+      showHide: false,
+    },
+  },
+  {
+    id: 'email',
+    isSortable: true,
+    cellActions: [
+      ({ rowIndex, columnId, Component }) => {
+        const row = ++rowIndex;
+        return (
+          <Component
+            onClick={() =>
+              alert(`Love sent from row ${row}, column "${columnId}"`)
+            }
+            iconType="heart"
+            aria-label={`Send love to ${row}, column "${columnId}" `}>
+            Send love
+          </Component>
+        );
+      },
+    ],
+  },
+  {
+    id: 'city',
+    isSortable: true,
+    cellActions: [
+      ({ rowIndex, columnId, Component, isExpanded }) => {
+        const row = ++rowIndex;
+        const message = isExpanded
+          ? `Cheers sent in Popover to row "${row}" column "${columnId}"`
+          : `Cheers sent from row ${row}, column "${columnId}"`;
+
+        return (
+          <Component
+            onClick={() => alert(message)}
+            iconType="cheer"
+            aria-label={message}>
+            Cheer
+          </Component>
+        );
+      },
+    ],
+  },
+  {
+    id: 'country',
+    cellActions: [
+      ({ rowIndex, columnId, Component }) => {
+        const row = ++rowIndex;
+        const label = `Send love to country at row ${row}, column "${columnId}"`;
+        return (
+          <Component
+            onClick={() =>
+              alert(`Love sent to country at row ${row}, column "${columnId}"`)
+            }
+            iconType="heart"
+            aria-label={label}>
+            Love this city
+          </Component>
+        );
+      },
+      ({ rowIndex, columnId, Component }) => {
+        const row = rowIndex++;
```

Comment:
```suggestion
        const row = ++rowIndex;
```
To start from 0 instead of 1

Reply:
Very, very true 👍 thx